### DISCUSSION TITLE
1.9.4-beta反馈

### DISCUSSION THREAD

**Discussion by xushengfeng** [xushengfeng/eSearch#83] [General]
反馈 1.9.4-beta
主要是多屏和屏幕缩放下是否正常
其他功能的错误也可以反馈

**Comment by iwvw** [comment]
1.9.5
低分屏，1080P，默认的显示太大了，能占满整个屏幕。
还有默认的快捷键也有些混乱，感觉不如snipaste清晰，有点无从下手的感觉
![image](https://user-images.githubusercontent.com/81206855/208812573-6173d157-89f7-4557-b36a-57d886c48ac3.png)

**Reply by xushengfeng** [reply]
可以在 设置-工具栏-按钮大小 里调节大小
默认快捷键没有几个，主要还是靠自己自定义，默认的<kbd>alt</kbd>+<kbd>c</kbd>是与其他聊天工具的截图键类似。软件除了截屏，还侧重ocr，所以框选好后按回车就是ocr。<kbd>esc</kbd>退出是通用的退出键。复制和保存也是通用的快捷键。所以大部分快捷键都是通用的。:)

**Reply by iwvw** [reply]
ok

**Comment by langseheise** [comment]
贴图时，图片会变得很大，分辨率不符合

**Comment by langseheise** [comment]
![Snipaste_2022-12-26_15-54-36](https://user-images.githubusercontent.com/43820174/209521395-56ad3017-de44-4567-a961-34015e368f53.png)
